install_config/aggregate_logging.adoc

```diff
@@ -1343,6 +1343,206 @@ This causes Fluentd to write logs to the Fluentd default location, *_/var/log/fl
 oc set env ds/fluentd LOGGING_FILE_PATH=console
 ----
 
+[[fluentd-merge-json]]
+*Configuring log collection JSON parsing*
+
+You can configure the Fluentd log collector to determine if a log message is in *JSON* format and merge
+the message into the JSON payload document posted to Elasticsearch. This feature is disabled by default.
```

Comment:
In 3.11, this feature is enabled by default.  These instructions should be geared towards those users who want to disable it because they are running into the problems listed below.